sig-ml-ops/README.md

```diff
@@ -0,0 +1,33 @@
+# ML Ops Special Interest Group
+
+SIG ML Ops covers the services and apis responsible for providing ML Ops capabilities to users of the Open Data Hub
+
+This document defines the scope and governance of the Model Ops Special Interest Group
```

Comment:
You have "Model Ops" here where we use "ML Ops" in other places. I think we should standardize on ML Ops.